src/models/migrate_user_request.rs

```diff
@@ -36,10 +39,16 @@ pub struct MigrateUserRequest {
     pub last_name: Option<String>,
     #[serde(rename = "properties", skip_serializing_if = "Option::is_none")]
     pub properties: Option<HashMap<String, Value>>,
+    #[serde(rename = "update_password_required")]
+    pub update_password_required: bool,
```

Comment:
This should be an optional field, with the same skip_serializing_if

and then you don't need it in the `new` constrcutor